lib/cucumber.rb

```diff
@@ -9,6 +9,7 @@
 module Cucumber
   class << self
     attr_accessor :wants_to_quit
+    attr_accessor :treats_ambiguous_as_fullwidth
```

Comment:
I think having this option is fine, but we need to add it to https://github.com/cucumber/cucumber-ruby/blob/master/lib/cucumber/configuration.rb instead.

This has implications for how we use it in the `DataTable`, but we can figure that out.